ES-MultiNode/es-ubuntu-install.sh

```diff
@@ -240,21 +239,35 @@ es_heap_size=$(free -m |grep Mem | awk '{if ($2/2 >31744)  print 31744;else prin
 printf "\nES_HEAP_SIZE=%sm\n" $es_heap_size | sudo tee -a /etc/default/elasticseach > /dev/null
 printf "MAX_LOCKED_MEMORY=unlimited\n" | sudo tee -a /etc/default/elasticsearch > /dev/null
 printf "\nelasticsearch - nofile 65536" | sudo tee -a /etc/security/limits.conf > /dev/null
-printf "\nelasticsearch - memlock unlimited" | sudo tee -a /etc/security/limits.conf > /dev/null
+printf "\nelasticsearch  - memlock unlimited" | sudo tee -a /etc/security/limits.conf > /dev/null
+sudo mkdir -p /etc/systemd/system/elasticsearch.service.d
+printf "\n[Service]\nLimitMEMLOCK=infinity\n" | sudo tee -a /etc/systemd/system/elasticsearch.service.d/elasticsearch.conf > /dev/null
 
-
-echo "#################### Starting services ####################"
+echo "#################### Start Elasticsearch service ####################"
+sudo systemctl daemon-reload
 sudo systemctl start elasticsearch.service
```

Comment:
Good question. Initially I thought installing x-pack requires ES service to be running, but now I doubt it. I am going to try to start ES and Kibana just once and see if it is sufficient